server/command/command.go

```diff
@@ -34,15 +34,39 @@ type RegisterFunc func(*model.Command) error
 
 type handleFunc func(parameters ...string) (string, bool, error)
 
+var cmds = []*model.AutocompleteData{
+	model.NewAutocompleteData("connect", "", "Connect to your Microsoft account"),
+	model.NewAutocompleteData("disconnect", "", "Disconnect from your Microsoft Account"),
+	model.NewAutocompleteData("summary", "", "View your events for today, or edit the settings for your daily summary."),
+	model.NewAutocompleteData("viewcal", "", "View your events for the upcoming week."),
+	model.NewAutocompleteData("settings", "", "Edit your user personal settings."),
+	model.NewAutocompleteData("subscribe", "", "Enable notifications for event invitations and updates."),
+	model.NewAutocompleteData("unsubscribe", "", "Disable notifications for event invitations and updates."),
+	model.NewAutocompleteData("autorespond", "[message]", "Set your auto-respond message."),
+	model.NewAutocompleteData("info", "", "Read information about this version of the plugin."),
+	model.NewAutocompleteData("help", "", "Read help text for the commands"),
+}
```

Comment:
@aaronrothschild Can I get your review on the order of these commands? Should be good to merge otherwise